common/djangoapps/student/helpers.py

```diff
@@ -35,18 +58,70 @@ def set_logged_in_cookie(request, response):
         expires_time = time.time() + max_age
         expires = cookie_date(expires_time)
 
-    response.set_cookie(
-        settings.EDXMKTG_COOKIE_NAME, 'true', max_age=max_age,
-        expires=expires, domain=settings.SESSION_COOKIE_DOMAIN,
-        path='/', secure=None, httponly=None,
-    )
+    cookie_settings = {
+        'max_age': max_age,
+        'expires': expires,
+        'domain': settings.SESSION_COOKIE_DOMAIN,
+        'path': '/',
+        'secure': None,
+        'httponly': None,
+    }
+
+    # Backwards compatibility: set the cookie indicating that the user
+    # is logged in.  This is just a boolean value, so it's not very useful.
+    # In the future, we should be able to replace this with the "user info"
+    # cookie set below.
+    response.set_cookie(settings.EDXMKTG_LOGGED_IN_COOKIE_NAME, 'true', **cookie_settings)
```

Comment:
Discussed offline: I'd like to remove this cookie, but I want to wait until after both edx-platform and the marketing site are updated.